docs/lib/content/using-npm/workspaces.md

```diff
@@ -102,7 +102,7 @@ the npm installer that package should be added as a dependency of the provided
 workspace:
 
 ```
-npm install abbrev -w a
+npm install abbrev -w packages/a
```

Comment:
This was actually correct, and included to show that the `-w` flag accepts the name of the workspace, not just the path.

Reply:
It's not working here (Windows 10, Node 18.15.0)

![image](https://github.com/npm/cli/assets/1009621/e1c56345-3403-4489-b68e-d7ed7b9d3912)

Reply:
You want `npm install abbrev -w my-budget`

Reply:
What is the `name` attribute of the `package.json` in `packages/app`?

Reply:
OK, I get it. The argument to `-w` is not the directory name but the package name defined in `package.json`. Sorry for the misunderstanding, but it's not precisely stated in the documentation. The example used in documentation is directory structure oriented, hence my confusion.